src/initialization/InitDistribution.H

```diff
@@ -40,8 +40,9 @@ namespace impactx::initialization
 
     /** Ignore the shape of a distribution and use the 2nd moments to create a covariance matrix
      */
-    CovarianceMatrix
-    create_covariance_matrix (
+    Envelope
+    create_envelope (
+        amrex::ParticleReal const current,
```

Comment:
If we want to make current/charge optional, we could make this the 2nd argument and use `std::optional`, which is similar to Python's `=None`.

Reply:
There is a small inconsistency with naming current/charge/intensity here that we can unify.

Reply:
updated